locale/en/textarea.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "",
+  "intro": "",
+  "note.title": "",
+  "note.items": ["", "", "", ""],
+  "sections": [
+    {
+      "title": "",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": ""
+        }
+      ],
+      "links": [
+        {
+          "text": "",
+          "url": ""
+        }
+      ]
+    },
+    {
+      "title": "",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": ""
+        }
+      ],
+      "links": [
+        {
+          "text": "",
+          "url": ""
+        }
+      ]
+    }
+  ],
+
+  "example.regular.label": "",
+  "example.regular.hintText": "",
+  "example.regular.placeholder": "",
+  "example.error.title": "",
+  "example.error.description": "",
+  "example.error.label": "",
+  "example.error.statusText": "",
```

Comment:
"example.error.hintText" seems to be missing

Reply:
Should now be there :)